packages/data-point/README.md

```diff
@@ -1211,6 +1211,34 @@ constant(value:*):*
   ```
 </details>
 
+
+<details>
+  <summary>constants that contain other reducers will not be evaluated</summary>
```

Comment:
this summary is a little misleading, since these constants are still "evaluated". not sure how to reword it though; maybe something like this?

`reducers are not evaluated when they're defined inside of constants`

Reply:
More specifically, no functions inside of a `constant` are evaluated, right?

Reply:
That's true, but this particular example doesn't have a function inside of a constant. If we do add that to the example (which wouldn't hurt), we still want to get across the fact that *any* reducer type inside a constant is not evaluated

Reply:
Cool, thanks for confirming.